src/behavior-property-observer.js

```diff
@@ -92,3 +117,51 @@ export class BehaviorPropertyObserver {
     this.removeSubscriber(context, callable);
   }
 }
+
+const numCons = Number;
+const dateCons = Date;
+const _isFinite = isFinite;
+const _isNaN = isNaN;
```

Comment:
It is expected that those function will be called with high frequency, so it is better to put them in the same scope with the calling function. This is to work around bundlers, which will add many scope layers containing maybe up to hundred of enumerable accessors.